dip-template/nodes/dip-consumer/src/service.rs

```diff
@@ -350,71 +339,70 @@ fn build_import_queue(
 	.map_err(Into::into)
 }
 
-#[allow(clippy::too_many_arguments)]
```

Comment:
I think we still need to ignore this...

Reply:
returned back: [a618f41](https://github.com/KILTprotocol/kilt-node/pull/627/commits/a618f41ead95fa33099e06c5c1df158d97b132e0)